scene/gui/control.cpp

```diff
@@ -73,6 +73,13 @@ Dictionary Control::_edit_get_state() const {
 }
 
 void Control::_edit_set_state(const Dictionary &p_state) {
+	ERR_FAIL_COND(p_state.size() <= 0);
+	ERR_FAIL_COND(p_state["rotation"].is_null());
+	ERR_FAIL_COND(p_state["scale"].is_null());
+	ERR_FAIL_COND(p_state["pivot"].is_null());
+	ERR_FAIL_COND(p_state["anchors"].is_null());
+	ERR_FAIL_COND(p_state["offsets"].is_null());
```

Comment:
Maybe we can chain the conditions with `||` to avoid having 6 separate checks for this unlikely error?

Reply:
Done